### PR TITLE
bugfix: LT-1845 - Notification Banner

### PR DESCRIPTION
**Pull request by andytaylorkainos**

### CODE REVIEW

File: cases/views/views.py [hunk 1]
```diff
@@ -60,7 +61,7 @@ def put(self, request, pk):
         Change the queues a case belongs to
         """
         case = get_case(pk)
-        serializer = CaseDetailSerializer(case, data=request.data, partial=True)
+        serializer = CaseUpdateSerializer(case, data=request.data, partial=True)
```
Comment (andytaylorkainos) [inline cases/views/views.py]: This is for the `put` request (not the `get` - github has condensed previous lines poorly so it looks like this is a change for the `get`)

File: gov_users/tests/tests_gov_user_notifications.py [hunk 2]
```diff
@@ -58,7 +58,23 @@ def test_edit_application_updates_previous_case_notification_success(self):
         self.assertEqual(data["name"], case_audit_notification.last().content_object.payload["new_name"])
         self.assertNotEqual(case_audit_notification.last().content_object, audit)
 
-    def test_get_case_notification_deletes_case_notification_and_returns_data_success(self):
+    def test_edit_application_as_gov_user_does_not_create_an_audit_notification_success(self):
```
Comment (andytaylorkainos) [inline gov_users/tests/tests_gov_user_notifications.py]: just renamed the test - didnt write the code (again github misinterpreting change)

File: gov_users/views/gov_user_views.py [hunk 3]
```diff
@@ -170,3 +170,13 @@ class UserMeDetail(APIView):
     def get(self, request):
         serializer = GovUserViewSerializer(request.user)
         return JsonResponse(data={"user": serializer.data})
+
+
+class NotificationViewSet(APIView):
```
Comment (andytaylorkainos) [inline gov_users/views/gov_user_views.py]: can remove for this PR, left as a convenience endpoint for another story in this sprint

File: cases/serializers.py [hunk 4]
```diff
@@ -213,6 +215,26 @@ def get_has_advice(self, instance):
         return has_advice
 
 
+class CaseDetailSerializer(CaseUpdateSerializer):
```
Comment (cstobart) [inline cases/serializers.py]: These seem like misnomers?

File: cases/views/views.py [hunk 5]
```diff
@@ -42,7 +43,7 @@
 
 
 class CaseDetail(APIView):
-    authentication_classes = (SharedAuthentication,)
+    authentication_classes = (GovAuthentication,)
```
Comment (cstobart) [inline cases/views/views.py]: Please run both sets of e2e's as changes to a SharedAuth endpoint

**Pull request merged**